src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -109,6 +110,9 @@ message ContainerResources {
     /// (Optional) Size of shared memory /dev/shm. If not set, a default value will be provided. A provided value
     // must be less than or equal to amount of memory allocated.
     uint32 shmSizeMB = 8;
+
+    /// (Optional) IP addresses allocated from Titus VPC IP service to be assigned to tasks.
+    repeated SignedAddressAllocation signedAddressAllocations = 9;
```

Comment:
We should not depend on `titus_vpc_api.proto`, as it is an agent side API. We could extract this message into `titus_base.proto`, to share it between different APIs.

Reply:
Yeah, that seems like a good suggestion.

Reply:
Adding @sargun since this package move may affect how these types are consumed by the VPC service.